#input should be the feature_index,feature_value and object_list
#request field
int64 feature_ind
float32[] feature_value
beginner_tutorials/Object[] object_list
---
#Output should be the object_index
int64[] outputobject

